Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsViewController_Tests.swift

```diff
@@ -0,0 +1,238 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatTestTools
+@testable import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerSuggestionsViewController_Tests: XCTestCase {
+    
+    // We need to provide a size to the suggestions view since here we are testing the view in isolation,
+    // and so we can't attach it to a bottomAnchorView. The test to verify the height calculation dependent
+    // on the rows should be done in the parent view controller tests.
+    private let defaultCommandSuggestionsSize = CGSize(width: 360, height: 210)
+    private let defaultMentionSuggestionsSize = CGSize(width: 360, height: 110)
```

Comment:
Why not use just one size for both? :)

Reply:
One had 4 commands, and the other and 2 mentions 😄 but since I'm going to use new mocked commands now, I can change that to be the same 👍